[BUG]: bootc image build broken do to upstream rpm-ostree manifest update

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

This [change](https://gitlab.com/fedora/bootc/base-images/-/commit/83ba3346ade9fc2fcad0bf09f9011119eb127307) in the upstream repository has borken the bootc image builds. The current AlmaLinux 10 repositories have bootc 1.8.0 package. It looks like the dracut bootc module was added to the bootc package at 1.11.0 in November. 

I'm not sure how you would like to fix this. Locally, I am going to vendor the rpm-ostree manifests instead of pulling them from the centos:stream10 image. 

Current error pulled from the build action

```
Creating empty machine-id
/usr/bin/dracut: line 1099: /sys/module/firmware_class/parameters/path: No such file or directory
dracut[E]: No '/dev/log' or 'logger' included for syslog logging
dracut[I]: Executing: /usr/bin/dracut --reproducible -v --add ostree --tmpdir=/tmp/dracut -f /tmp/initramfs.img --no-hostonly --kver 6.12.0-124.45.1.el10_1.x86_64
dracut[I]: 62bluetooth: Could not find any command of '/usr/lib/bluetooth/bluetoothd /usr/libexec/bluetooth/bluetoothd'!
dracut[E]: Module 'bootc' cannot be found.
error: Finalizing rootfs: During kernel processing: bwrap(rpmostree-dracut-wrapper): Child process exited with code 1
error: Executing compose install: ExitStatus(unix_wait_status(256))
Error executing command: Command '['rpm-ostree', 'compose', 'rootfs', '--source-root=/', '/usr/share/doc/bootc-base-imagectl/manifests/almalinux-10.yaml', '/target-rootfs']' returned non-zero exit status 1.
Error: building at STEP "RUN /usr/libexec/bootc-base-imagectl build-rootfs --reinject --manifest=almalinux-10 /target-rootfs": while running runtime: exit status 1
make: *** [Makefile:14: image] Error 1
```

### Expected Behavior

Expect the bootc image build to succeed. 

### Steps To Reproduce

No need to reproduce, the github actions nightly builds are actively reproducing the issue: https://github.com/AlmaLinux/bootc-images/actions/runs/23882992575/job/69639869540

### Anything else?

_No response_

### Search terms

dracut bootc

--- a/10/almalinux-10.yaml
+++ b/10/almalinux-10.yaml
@@ -24,6 +24,15 @@ postprocess:
   - |
     #!/usr/bin/env bash
     set -euo pipefail
+    # Remove 'bootc' dracut module from initramfs config
+    # AlmaLinux bootc package does not yet ship the dracut module (requires >= 1.11.0)
+    DRACUT_CONF="/usr/lib/dracut/dracut.conf.d/20-bootc-base.conf"
+    if [ -f "${DRACUT_CONF}" ]; then
+      sed -i 's/ bootc / /' "${DRACUT_CONF}"
+    fi
+  - |
+    #!/usr/bin/env bash
+    set -euo pipefail
     dnf clean all
     rm /var/{log,cache,lib}/* -rf
     systemctl preset-all

--- a/9/almalinux-9.yaml
+++ b/9/almalinux-9.yaml
@@ -24,6 +24,15 @@ postprocess:
   - |
     #!/usr/bin/env bash
     set -euo pipefail
+    # Remove 'bootc' dracut module from initramfs config
+    # AlmaLinux bootc package does not yet ship the dracut module (requires >= 1.11.0)
+    DRACUT_CONF="/usr/lib/dracut/dracut.conf.d/20-bootc-base.conf"
+    if [ -f "${DRACUT_CONF}" ]; then
+      sed -i 's/ bootc / /' "${DRACUT_CONF}"
+    fi
+  - |
+    #!/usr/bin/env bash
+    set -euo pipefail
     dnf clean all
     rm /var/{log,cache,lib}/* -rf
     systemctl preset-all

